backend/tests/session.test.ts

```diff
@@ -3,20 +3,9 @@ import request from 'supertest';
 import { Server } from 'http';
 import { Archive } from 'opentok';
 import InMemorySessionStorage from '../storage/inMemorySessionStorage'; // Import the session storage
+import mockOpentokConfig from '../helpers/__mocks__/config';
 
-await jest.unstable_mockModule('../helpers/config', () => {
-  return {
-    default: jest.fn().mockImplementation(() => {
-      return {
-        apiKey: 'test-api-key',
-        apiSecret: 'test-api-secret',
-        applicationId: 'test-application-id',
-        privateKey: 'test-private-key',
-        provider: 'opentok',
-      };
-    }),
-  };
-});
+await jest.unstable_mockModule('../helpers/config', mockOpentokConfig);
 
 const mockVcrSessionStorage = {
```

Comment:
I actually believe a lot of this test suite to not be doing what it looks like its doing, but I don't want to do everything in this PR